examples/poisson_solver/poisson_solver.cpp

```diff
@@ -231,8 +231,7 @@ int main(int argc, char *argv[])
                        gko::stop::ResidualNormReduction<>::build()
                            .with_reduction_factor(1e-6)
                            .on(exec))
-        // something fails here:
-        // .with_preconditioner(bj::create(exec, 32))
+        .with_preconditioner(bj::build().on(exec))
```

Comment:
Maybe you already mentioned it before, but was curious, what was the the problem here ?

Reply:
It was #128. We just forgot to uncomment the preconditioner lines of the examples in that PR.